foundation/foundation-mda/src/main/resources/aiops-record-schema.json

```
@@ -78,6 +78,29 @@
 					"name", "type"
 				]
 			}
+		},
+		"relations": {
+			"type": "array",
+			"items": {
+				"type": "object",
+				"properties": {
+					"type": {
+						"type": "string"
+					},
+					"package": {
+						"type": "string"
+					},
+					"documentation": {
+						"type": "string"
+					},
+					"multiplicity": {
+						"type": "string"
+					}
+				},
+				"required": [
+					"name","type"
```

S:
```suggestion
					"package","type"
```
or if we align more with the step record type specification
```suggestion
					"name","package"
```

---

I'll align with the step record type i.e. "name","package"

- [x] Update schema json with required: name and package